src/components/layout/nav.js

```diff
@@ -1089,14 +1089,12 @@ export const NavPartners = ({ no_login_signup }) => {
                                 src={Close}
                                 alt="close menu 2"
                                 onClick={closeOffCanvasMenu}
-                                width="16px"
```

Comment:
Instead of deleting this, you can specify it like `width={16}` or have the width set in `CloseMenuPartners` styles above itself. This is was a breaking change when we upgraded to Preact and it doesn't accept pixels for inline width and height.

Reply:
There is not any difference between passing 16 and nothing, because of that I removed it